core/templates/dev/head/pages/topic_editor/TopicEditorStateService.js

```diff
@@ -40,10 +40,11 @@ oppia.factory('TopicEditorStateService', [
     var _topic = TopicObjectFactory.createInterstitialTopic();
     var _topicRights = TopicRightsObjectFactory.createInterstitialRights();
     // The array that caches all the subtopic pages loaded by the user.
-    var _subtopicPages = [];
-    // The array that stores all the subtopic pages not loaded from the backend
-    // i.e those that correspond to newly created subtopics.
-    var _newSubtopicPages = [];
+    var _cachedSubtopicPages = [];
+    // The array that stores all the ids of the subtopic pages that were not
```

Comment:
... that were newly-created (and not loaded from the backend).

(Reasoning: the way you've written it, it implies that any subtopic pages that were not loaded from the backend are those that are cached in the frontend. This isn't quite right; it ignores the case of pages that exist but haven't been requested yet.)

Reply:
Done.